wuzz.go

```diff
@@ -149,7 +133,12 @@ func (e *ViewEditor) Edit(v *gocui.View, key gocui.Key, ch rune, mod gocui.Modif
 		e.backTabEscape = true
 		return
 	}
-
+	if key == gocui.KeyHome {
+		_ = e.GoToHome(v)
```

Comment:
`_ = ` is not necessary.
Another thing: isn't `return` required after these statements? Without returning the original editor function will be called as well. Alternatively it can be turned into a switch-case statement like this:
```golang
switch key {
case gocui.KeyHome:
    e.GoToHome(v)
default:
    e.origEditor.Edit(..)
```